soc/microchip/mec/Kconfig

```diff
@@ -210,6 +211,54 @@ config MCHP_HEADER_VERBOSE_OUTPUT
 
 endif # MCHP_MEC_UNSIGNED_HEADER
 
+# Common debug configuration
```

Comment:
Maybe is worth consolidate similar options for MEC152x/MEC172x (non-hal) using simply this KConfig. 
- /soc/arm/microchip_mec/mec172x/Kconfig.soc:    config SOC_MEC172X_DEBUG_WITHOUT_TRACING

However, I believe that is beyond this PR scope not related to MEC175x also need to deprecate existing ones first, 
 which affect existing customer